news/2024/2024-11-14-osu-world-cup-2024-round-of-16-recap.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title: "osu! World Cup 2024: Round of 16 Recap"
+date: 2024-11-14 16:00:00 +0000
+---
+
+The first OWC elimination round has concluded — who survived, and who's going home? Tune in to find out!
+
+![](/wiki/Tournaments/OWC/2024/img/owc2024-banner.jpg)
+
+This past weekend marked the start of the first elimination round, and ended with eight fewer teams competing for the main prize. The show must go on though, so let's jump right into the Round of 16 recap!
+
+You can also check out the high-effort video coverage by our own [Junihuhn](https://osu.ppy.sh/users/4182339) below!
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Q3t9MojGc18" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Mappool recap
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+**[Download the pool here! (131 MB)](https://osu.ppy.sh/beatmaps/packs/P265)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2292992412&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![](/wiki/shared/news/2024-11-14-osu-world-cup-2024-round-of-16-recap/owc-2024-ro16-mappool.jpg)
+
+This week's map pool had a unique twist, with some of the picks being throwbacks to world cups of the past! [ChaiN De/structioN](https://osu.ppy.sh/beatmapsets/49071/#osu/154165) by [Shiirn](https://osu.ppy.sh/users/465126) (DT2) and [bmin11](https://osu.ppy.sh/users/43468)'s [Silent Voyage to Eternity](https://osu.ppy.sh/beatmapsets/14309/#osu/115384) (FM2) sparked nostalgia for many of the older players, with ChaiN De/structioN getting pooled for the first time since 2012's [OWC #3](/wiki/Tournaments/OWC/3), and serving as speed challenge with a focus on bursts. Silent Voyage to Eternity, last seen in [OWC 2016](/wiki/Tournaments/OWC/2016)'s finals stage as a Hidden pick, served as both a testament to how much players have improved since then, and a reminder that some old, remarkable scores by Chinese legend [rustbell](https://osu.ppy.sh/users/227717) still very much [stand the test of time](/wiki/shared/news/2024-11-14-osu-world-cup-2024-round-of-16-recap/rustbell-old-score.jpg).
```

Comment:
```suggestion
This week's map pool had a unique twist, with some of the picks being throwbacks to world cups of the past! [ChaiN De/structioN](https://osu.ppy.sh/beatmapsets/49071/#osu/154165) by [Shiirn](https://osu.ppy.sh/users/465126) (DT2) and [bmin11](https://osu.ppy.sh/users/43468)'s [Silent Voyage to Eternity](https://osu.ppy.sh/beatmapsets/14309/#osu/115384) (FM2) sparked nostalgia for many of the older players, with ChaiN De/structioN getting pooled for the first time since 2012's [OWC #3](/wiki/Tournaments/OWC/3), serving as a speed challenge with a focus on bursts. Silent Voyage to Eternity, last seen in [OWC 2016](/wiki/Tournaments/OWC/2016)'s finals stage as a Hidden pick, served as both a testament to how much players have improved since then, and a reminder that some old, remarkable scores by Chinese legend [rustbell](https://osu.ppy.sh/users/227717) still very much [stand the test of time](/wiki/shared/news/2024-11-14-osu-world-cup-2024-round-of-16-recap/rustbell-old-score.jpg).
```